bootstrap.py

```diff
@@ -174,12 +167,18 @@ def exec_failexit(command):
     print_success(command)
     print ""
 
+def yum(command, pkgs = ""):
+    for x in range(0, int(options.download)):
+        if exec_failok("/usr/bin/yum -y --downloadonly %s %s" % (command, pkgs)):
```

Comment:
`--downloadonly` is not always available (it was provided by an optional plugin in older yum versions)